### ISSUE TITLE
support alt text on exhibition objects images: 2

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#329]
The custom exhibition content type includes an image. We need to make changes for accessibility support:

## New Alt text property

 - Add an "alt text" field to the content type so that users can add and store the appropriate text. This field should be **required** and it cannot be zero-length.

## HTML markup changes

Currently, the HTML markup around the image is as follows:

```html
<div class="ExhibitObjectImage">
    <img src="http://isaw.nyu.edu/exhibitions/time-cosmos/objects/quadruple-vertical-sundial/@@images/769d56f3-a0b4-4891-9063-fb328c9686b7.jpeg" 
         alt="Quadruple Vertical Sundial with Inscription" 
         title="Quadruple Vertical Sundial with Inscription" 
         width="768" height="486">
</div>
```

The values for both the ```alt``` and ```title``` attributes seem to be coming from the item Title property.

Change the behavior so we get the following:

```html
<div class="ExhibitObjectImage">
    <img src="http://isaw.nyu.edu/exhibitions/time-cosmos/objects/quadruple-vertical-sundial/@@images/769d56f3-a0b4-4891-9063-fb328c9686b7.jpeg" 
         alt="Whatever the author put in the 'alt text' field on the content item"
         width="768" height="486">
</div>
```

That is:

 - suppress the ```title``` attribute entirely
 - fill the ```alt``` attribute from the new "alt text" field on the object

**Comment by cguardia** [comment]
This is on staging now.
